code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -727,3 +727,52 @@
 
 	new /obj/item/analyzer(src)
 	new /obj/item/healthanalyzer(src)
+
+/obj/item/storage/belt/mining
+	name = "explorer's webbing"
+	desc = "A versatile chest rig, cherished by miners and hunters alike."
+	icon_state = "explorer1"
+	item_state = "explorer1"
+	storage_slots = 6
+	max_w_class = WEIGHT_CLASS_BULKY
+	use_item_overlays = 0
+	can_hold = list(
+		/obj/item/crowbar,
+		/obj/item/screwdriver,
+		/obj/item/weldingtool,
+		/obj/item/wirecutters,
+		/obj/item/wrench,
+		/obj/item/multitool,
+		/obj/item/flashlight,
+		/obj/item/stack/cable_coil,
+		/obj/item/analyzer,
+		/obj/item/extinguisher/mini,
+		/obj/item/radio,
+		/obj/item/clothing/gloves,
+		/obj/item/resonator,
+		/obj/item/mining_scanner,
+		/obj/item/pickaxe,
+		/obj/item/stack/sheet/animalhide,
+		/obj/item/stack/sheet/sinew,
+		/obj/item/stack/sheet/bone,
+		/obj/item/lighter,
+		/obj/item/storage/fancy/cigarettes,
+		/obj/item/reagent_containers/food/drinks/bottle,
+		/obj/item/stack/medical,
+		/obj/item/kitchen/knife,
+		/obj/item/reagent_containers/hypospray,
+		/obj/item/gps,
+		/obj/item/storage/bag/ore,
+		/obj/item/survivalcapsule,
+		/obj/item/t_scanner/adv_mining_scanner,
+		/obj/item/reagent_containers/food/pill,
+		/obj/item/stack/ore,
+		/obj/item/reagent_containers/food/drinks,
+		/obj/item/organ/internal/hivelord_core,
+		/obj/item/wormhole_jaunter,
+		/obj/item/storage/bag/plants,
+		/obj/item/stack/marker_beacon)
+
+/obj/item/storage/belt/mining/New()

```

Comment:
I believe they made it so only the vendor version comes with capsules?

Reply:
Yep, the only other way to get it though is from a dead legion corpse, so it's not very significant.